docs/BrewTestBot-For-Maintainers.md

```diff
@@ -32,3 +32,13 @@ If a pull request needs its commit messages changed in a way that autosquash doe
 1. Ensure that bottles have built successfully.
 2. Run `brew pr-pull 12345` where `12345` is the pull request number (or URL).
 3. Amend any relevant commits if needed, then run `git push` to push the commits to the pull request.
+
+## Rebottling
+
+If a formula needs rebottling for any reasons within homebrew-core:
+
+1. Navigate to the homebrew-core github page.
```

Comment:
Before @issyl0 sees it:

```suggestion
1. Navigate to the homebrew-core GitHub page.
```

Reply:
Or, even better, I guess we can just replace the first 3 points with something that includes the URL so that it's easier to follow. Perhaps something like:

> Navigate to homebrew-core's rebottling workflow page at https://github.com/Homebrew/homebrew-core/actions/workflows/dispatch-rebottle.yml